### ISSUE TITLE
TransT-GOT Train Setting?

### ISSUE THREAD

**Issue by SkiddieAhn** [chenxin-dlut/TransT#128]
I want to know the learning settings of TransT-Got. 
1. batch size
2. samples per epoch
3. epoch
4. learning rate & weight decay

And I want to know if I can use the 'sampler for training' code like this.

 # The sampler for training
 dataset_train = sampler.TransTSampler([got10k_train], None,
                                samples_per_epoch=1000*settings.batch_size, max_gap=100, processing=data_processing_train)

**Comment by cmj18** [comment]
samples per epoch等于多少。。理论上来说应该是GOT10K-train的9335个视频序列吧
